frontend/src/common/js/javascript.js

```diff
@@ -95,38 +95,38 @@ CodeMirror.defineMode("javascript", function(config, parserConfig) {
         stream.eat("=");
         return ret("operator", "operator", stream.current());
       }
-    } else if (ch == "`") {
+    } else if (ch === "`") {
       state.tokenize = tokenQuasi;
       return tokenQuasi(stream, state);
-    } else if (ch == "#" && stream.peek() == "!") {
+    } else if (ch === "#" && stream.peek() === "!") {
       stream.skipToEnd();
       return ret("meta", "meta");
-    } else if (ch == "#" && stream.eatWhile(wordRE)) {
+    } else if (ch === "#" && stream.eatWhile(wordRE)) {
       return ret("variable", "property")
-    } else if (ch == "<" && stream.match("!--") ||
-               (ch == "-" && stream.match("->") && !/\S/.test(stream.string.slice(0, stream.start)))) {
+    } else if (ch === "<" && stream.match("!--") ||
+               (ch === "-" && stream.match("->") && !/\S/.test(stream.string.slice(0, stream.start)))) {
       stream.skipToEnd()
       return ret("comment", "comment")
     } else if (isOperatorChar.test(ch)) {
-      if (ch != ">" || !state.lexical || state.lexical.type != ">") {
+      if (ch !== ">" || !state.lexical || state.lexical.type !== ">") {
         if (stream.eat("=")) {
-          if (ch == "!" || ch == "=") stream.eat("=")
+          if (ch === "!" || ch === "=") stream.eat("=")
         } else if (/[<>*+\-|&?]/.test(ch)) {
           stream.eat(ch)
-          if (ch == ">") stream.eat(ch)
+          if (ch === ">") stream.eat(ch)
         }
       }
-      if (ch == "?" && stream.eat(".")) return ret(".")
+      if (ch === "?" && stream.eat(".")) return ret(".")
       return ret("operator", "operator", stream.current());
     } else if (wordRE.test(ch)) {
       stream.eatWhile(wordRE);
       var word = stream.current()
-      if (state.lastType != ".") {
+      if (state.lastType !== ".") {
         if (keywords.propertyIsEnumerable(word)) {
           var kw = keywords[word]
           return ret(kw.type, kw.style, word)
         }
-        if (word == "async" && stream.match(/^(\s|\/\*([^*]|\*(?!\/))*?\*\/)*[\[\(\w]/, false))
+        if (word === "async" && stream.match(/^(\s|\/\*([^*]|\*(?!\/))*?\*\/)*[\[\(\w]/, false))
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/riskscanner/riskscanner/pullRequest?prid=7134172)
```suggestion
        if (word === "async" && stream.match(/^(\s|\/\*([^*]|\*(?!\/))*?\*\/)*[\[\(\w]/, false))
```